components/Recipe/index.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import styled from "styled-components";
+
+const RecipeContainer = styled.div`
+  flex-basis: calc(50% - 10px);
+  display: flex;
+  flex-direction: column;
+`;
+
+const RecipeImage = styled.img`
+  width: 100%;
+  height: 200px;
+  object-fit: cover;
+  border-radius: 15px;
+`;
```

Comment:
This should be an extended next Image like this:

```suggestion
import Image from "next/image";

const RecipeImage = styled(Image)`
  width: 100%;
  height: 200px;
  object-fit: cover;
  border-radius: 15px;
`;
```

Reply:
I'll adjust it, thanks